### PR TITLE
[CVP-2977] Upgrade operator-sdk to v1.26.0 for operator bundle testing

### PR DESCRIPTION
**Pull request by asergienk**
Downstream changes will be merged first

From CVP-2977 deliverables, this MR takes care of the highlighted ones:

- Update the operator-utils container to use operator-sdk >= v1.26.0 and add the new images to the image source test allow-list
- Run the operator-bundle-retest pipeline (skip deploy) on the MR to the cvp/pipeline repository
- Update the operator-test-playbooks to use operator-sdk >= v1.26.0
- **Update the operator-scorecard-test-container image to use operator-sdk >= v1.26.0**
- Build and push the updated operator-scorecard-test-container image to the quay.io/operator-framework/scorecard-test repository

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM registry.fedoraproject.org/fedora:32
```
Comment (nmars) [inline Dockerfile:1]: I think we should update to fedora:37

Comment (asergienk) [reply Dockerfile]: done

**Pull request merged**